bigquery_etl/cli/glean_usage.py

```diff
@@ -0,0 +1,106 @@
+"""bigquery-etl CLI glean_usage command."""
+from functools import partial
+from multiprocessing.pool import ThreadPool
+import click
+
+from ..cli.utils import (
+    is_valid_project,
+    table_matches_patterns,
+)
+from ..glean_usage import (
+    baseline_clients_daily,
+    baseline_clients_first_seen,
+    baseline_clients_last_seen,
+)
+from ..glean_usage.common import list_baseline_tables, get_app_info
+
+# list of methods for generating queries
+GLEAN_TABLES = [
+    baseline_clients_daily.BaselineClientsDailyTable(),
+    baseline_clients_first_seen.BaselineClientsFirstSeenTable(),
+    baseline_clients_last_seen.BaselineClientsLastSeenTable(),
+]
+
+
+@click.group(help="Commands for managing Glean usage.")
+def glean_usage():
+    """Create the CLI group for the glean_usage command."""
+    pass
+
+
+@glean_usage.command()
+@click.option(
+    "--project-id",
+    "--project_id",
+    help="GCP project ID",
+    default="moz-fx-data-shared-prod",
+    callback=is_valid_project,
+)
+@click.option(
+    "--output-dir",
+    "--output_dir",
+    help="Output directory generated SQL is written to",
```

Comment:
To make this consistent with use elsewhere in the CLI, we should include `project_id` in the generated paths under output_dir. Currently, we have to pass in the project-level directory to this option, which is surprising. It seems like this refactor is the right place to fix this problem.